docs/user-guide/ze-usage.md

```diff
@@ -0,0 +1,287 @@
+2. Click **Refresh All** button on the right of the **DATA SETS** explorer bar.
+
+#### Rename data sets
+
+1. Navigate to the explorer tree.
+2. Open the **DATA SETS** bar.
+3. Select a data set you want to rename.
+4. Right-click the data set and select the **Rename Data Set** option.
+5. Change the name of the data set.
+
+<img src="../images/ze/ZE-rename.gif" alt="Rename Data Set">
+
+#### Copy data sets
+
+1. Navigate to the explorer tree.
+2. Open the **DATA SETS** bar.
+3. Select a member you want to copy.
+4. Right-click the member and select the **Copy Data Set** option.
+5. Right-click the data set where the member belongs and select the **Paste Data Set** option.
+6. Enter the name of the copied member.
+   <img src="../images/ze/ZE-copy.gif" alt="Copy Data Set">
+
+#### Download, edit, and upload existing PDS members
+
+1. Navigate to the explorer tree.
+2. Open the **DATA SETS** bar.
+3. Open a profile.  
+4. Click the PDS member (or PS) that you want to download.
+
+    **Note:** To view the members of a PDS, click the PDS to expand the tree.
+
+    The PDS member displays in the text editor window of VSC.
+5. Edit the document.
+6. Navigate back to the PDS member (or PS) in the explorer tree, and click the **Save** button.
+
+Your PDS member (or PS) is uploaded.  
+
+**Note:** If someone else has made changes to the PDS member (or PS) while you were editing it, you can merge your conflicts before uploading to the mainframe.
+
+<img src="../images/ze/ZE-download-edit.gif" alt="Edit">
+
+#### Use the save option to prevent merge conflicts
+
+1. Navigate to the explorer tree.
+2. Open the **DATA SETS** bar.
+3. Open a profile.
+4. Download and edit a data set.
+5. Click the **Save** button for the data set that you opened in the explorer tree.
+6. Resolve merge conflicts if necessary.
+
+<img src="../images/ze/ZE-safe-save.gif" alt="Save">
+
+#### Create a new PDS and a PDS member
+
+1. Navigate to the explorer tree.
+2. Open the **DATA SETS** bar.
+3. Click the **Create New Data Set** button to specify the profile that you want to create the data set with.
+4. From the drop-down menu, select the type of PDS that you want to create.
+5. Enter a name for the PDS.
+   The PDS is created.
+6. To create a member, right-click the PDS and select **Create New Member**.
+7. Enter a name for the member.
+   The member is created.
+
+<img src="../images/ze/ZE-cre-pds-member.gif" alt="Create">
+
+#### Delete a PDS member and PDS
+
+1. Navigate to the explorer tree.
+2. Open the **DATA SETS** bar.
+3. Open the profile and PDS containing the member.
+4. Right-click on the PDS member that you want to delete and select **Delete Member**.
+5. Confirm the deletion by clicking **Yes** on the drop-down menu.
+
+    **Note:** Alternatively, you can select 'No' to cancel the deletion.
+6. To delete a PDS, right-click the PDS and click **Delete PDS**, then confirm the deletion.
+
+    **Note:** You can delete a PDS before you delete its members.
```

Comment:
What happens to the members? I feel like there is an issue open for this behavior...

Reply:
As far as I know, all members of that PDS will be deleted, too.